mapie/regression.py

```diff
@@ -84,24 +85,25 @@ class MapieRegressor(BaseEstimator, RegressorMixin):  # type: ignore
         If ``-1`` all CPUs are used.
         If ``1`` is given, no parallel computing code is used at all,
         which is useful for debugging.
-        For n_jobs below ``-1``, ``(n_cpus + 1 + n_jobs)`` are used.
+        For n_jobs below ``-1``, ``(n_cpus + 1 - n_jobs)`` are used.
         None is a marker for ‘unset’ that will be interpreted as ``n_jobs=1``
         (sequential execution).
 
         By default ``None``.
 
-    ensemble: bool, optional
-        Determines how to return the predictions.
-        If ``False``, returns the predictions from the single estimator
+    agg_function : str
+        Determines if and how the final prediction is aggregated from the fold
+        predictions. In the case of the Jackknife+-after-Bootstrap method, this
+        function is also used to aggregate the insample predictions.
+        If ``None``, returns the predictions from the single estimator
```

Comment:
Mentionner également que pour la méthode J+aB `agg_function = "mean"` si `None` ?

Reply:
j'ai mis une phrase

Reply:
Phrase modifiée pour être plus précis :
When the cross-validation strategy is Resample (e.g. for the
        Jackknife+-after-Bootstrap method), if ``None``, the
        default aggregation function to compute residauls is "mean".